pyproject.toml

```diff
@@ -36,6 +37,10 @@ classifiers = [
     "Topic :: Software Development :: Code Generators"
 ]
 
+[project.scripts]
+openassetio-traitgen = "openassetio_traitgen.__main__:main"
+
+
```

Comment:
[Consider][minor]
```suggestion
```

Reply:
Caught it before you! [175b224](https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/60/commits/175b224b656d99102b97bb3553f1b50bbee58a72)